app/main.ipynb

```diff
@@ -612,25 +1847,53 @@
    "metadata": {
     "collapsed": false,
     "ExecuteTime": {
-     "end_time": "2023-07-03T14:14:16.230395Z",
-     "start_time": "2023-07-03T14:14:16.206501Z"
+     "end_time": "2024-11-07T07:38:04.356013Z",
+     "start_time": "2024-11-07T07:38:04.349482Z"
     }
-   }
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "([-6, -3, -8, -5, -10], [1, 2, 1, 0, 8], [7, 6, 7, 12, 11])"
+      ]
+     },
+     "execution_count": 23,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "execution_count": 23
   },
   {
    "cell_type": "code",
-   "execution_count": 32,
-   "outputs": [],
    "source": [
-    "# write your numpy code here"
+    "array1 = np.array([5, 2, 7, 4, 9])\n",
+    "array2 = np.array([3, 6, 1, 8, 10])\n",
+    "~array1, array1 & array2, array1 | array2"
```

Comment:
The use of NumPy for bitwise operations on arrays is correct for the NumPy implementation.